ui/raidboss/emulator/data/CombatantTracker.ts

```diff
@@ -116,6 +91,7 @@ export default class CombatantTracker {
     initState.maxHp = initState.maxHp ?? extractedState.maxHp;
     initState.mp = initState.mp ?? extractedState.mp;
     initState.maxMp = initState.maxMp ?? extractedState.maxMp;
+    initState.targetable = initState.targetable ?? extractedState.targetable ?? true;
```

Comment:
I'm not sure I understand the additions to combatant tracker.  Are these a different feature vs a performance optimization?

Reply:
This is more of a bugfix. The property was already being tracked elsewhere but was missing being set here.

It could be done in a separate PR, I guess?

Reply:
Sorry to be pedantic, but by "this" in "this is more of a bugfix", do you mean only this line? By "the additions to combatant tracker", I meant this line and the other changes to this file.  Are those a different feature as well?  Sorry for the questions, but there was just a bunch of different changes in this PR and it's hard for me to back solve what the motivation for each of them was.

Reply:
This addition on line 94 that you've highlighted above is specifically a bugfix.

The other changes to this file are to facilitate only looping over log lines once instead of having to loop over them twice.